src/Azure/Orleans.Streaming.EventHubs/Providers/Streams/EventHub/IEventHubReceiver.cs

```diff
@@ -56,13 +56,8 @@ EventPosition GetEventPosition()
                 // If we have a starting offset, read from offset
                 if (offset != EventHubConstants.StartOfStream)
                 {
-                    if (!long.TryParse(offset, out var longOffset))
-                    {
-                        throw new InvalidOperationException("Offset must be a number.");
-                    }
-
                     LogInfoStartingRead(logger, options.EventHubName, partitionSettings.Partition, offset);
-                    eventPosition = EventPosition.FromOffset(longOffset, true);
+                    eventPosition = EventPosition.FromOffset(offset, true);
```

Comment:
Are we sure that we want this to be inclusive?   That is going to cause a duplicate to be read, as you'll replay the event with that offset.   I'm assuming that event was processed before a checkpoint was written.

Reply:
Yes, subscriber cursors need to start from an already delivered offset, to make sure we haven't lost anything